clearwater-diags-monitor/usr/share/clearwater/infrastructure/install/clearwater-diags-monitor.postinst

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# @file clearwater-diags-monitor.postinst
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2013  Metaswitch Networks Ltd
```

Comment:
Since I've simply extracted code from another file, I think we should leave it with the same copyright year as it already was.